### DISCUSSION TITLE
Android 15 MD changes

### DISCUSSION THREAD

**Discussion by laurentpayot** [beercss/beercss#387] [Ideas]
Hi, Android 15 introduced some MD changes: https://developer.android.com/about/versions/15/behavior-changes-15#ux

Do you think we could / should apply these changes to Beercss too? ([Edge-to-edge by default](https://developer.android.com/about/versions/15/behavior-changes-15#edge-to-edge) is nice)

**Comment by leonardorafael** [comment]
I don't read yet, but probably yes
